docs/integrations/howto/datadog-metrics.rst

```diff
@@ -0,0 +1,42 @@
+Send metrics to DataDog
+=======================
+
+This guide will show you how to send metrics from your Aiven service to your DataDog account.
+
+Prepare your DataDog account
+----------------------------
+
+You will need a DataDog account and the following details handy:
+
+* Which region you want to use, US or EU
+
+* An API key for this integration. Generate an API key by visiting **Organization settings** under your account menu, and then choose **API Keys**. The **New Key** button will give you an API key; you should copy this as you will need it shortly.
```

Comment:
As in previous article, let's be specific that we're talking about "A DataDog API key."

Reply:
Good catch! Fixed.